### DISCUSSION TITLE
Peak Annotation on Integrated Dataset

### DISCUSSION THREAD

**Discussion by bhekmat** [stuart-lab/signac#1529] [Q&A] [answered]
@timoast 

I used ChIPseeker to annoate the peaks from an integrated dataset containing WT and KO cells. Does Signac package have a function that would allow me to classify/subset the peaks by the cell types? Or could you suggest a way to achieve this?

Thank you.

**Comment by timoast** [comment, marked as answer]
You can perform differential accessibility testing to find peaks that are specific to different cell populations. Please see the vignettes: https://stuartlab.org/signac/articles/overview
